vipps-report-api-faq.md

```diff
@@ -36,6 +35,29 @@ API before summer 2023.
 Currently we only support single-MSN API keys (see the question below
 on API keys for more information on future plans for partner keys).
 
+## What are the benefits of the Report API over the SFTP service?
+
+The
+[Vipps SFTP Report Service](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/settlements/sftp-report-service)
+lets merchants and partners download settlement files by SFTP.
+
+The Report API has some benefits over this:
+* Data can be retrieved at any time.
+* Data is available regardless of whether the balance is positive or negative,
+  so even if there has only been refunds there will be data available.
+* The SFTP service only provides settlement files if the balance is positive.
+* It is possible to retrieve data for more than one day in one request, so
```

Comment:
I didn't like using "it is possible" or "in one request" twice in the same sentence. You could perhaps say: "It is possible to retrieve data for more than one day in a request, so you could, for example, retrieve an entire weekend, week, or month."

Reply:
Thanks, fixed!